api/admin/controller/library_settings.py

```diff
@@ -337,18 +331,37 @@ def list_setting(self, setting, json_objects=False):
 
         return json.dumps([_f for _f in value if _f])
 
-    def image_setting(self, setting):
-        """Retrieve an uploaded image for the given setting from the current HTTP request."""
+    @staticmethod
+    def _data_url_for_image(image: Image, _format="PNG") -> Optional[str]:
```

Comment:
I think to match the type hints to the comments and function behaviour this one should be `-> str`